heka/files/heka.service

```diff
@@ -34,6 +34,7 @@ respawn
 pre-start script
     touch /var/log/{{ service_name }}.log
     chown heka:heka /var/log/{{ service_name }}.log
+    chown -R heka:heka /var/cache/{{ service_name }}
```

Comment:
Why did you add this? Have you found an actual problem? If so I think a separate PR should be created.